client/main.lua

```diff
@@ -290,23 +297,26 @@ RegisterNetEvent('qb-tow:client:TowVehicle', function()
                     local towPos = GetEntityCoords(vehicle)
                     local targetPos = GetEntityCoords(targetVehicle)
                     if #(towPos - targetPos) < 11.0 then
-                        QBCore.Functions.Progressbar("towing_vehicle", Lang:t("mission.towing_vehicle"), 5000, false, true, {
-                            disableMovement = true,
-                            disableCarMovement = true,
-                            disableMouse = false,
-                            disableCombat = true,
-                        }, {
-                            animDict = "mini@repair",
-                            anim = "fixing_a_ped",
-                            flags = 16,
-                        }, {}, {}, function() -- Done
+                        if lib.progressBar({
```

Comment:
Was there a reason for removing the disable movement, mouse, and combat?

Reply:
Yeah, the fact that you have to be in the vehicle in order to attach the car.

![image](https://github.com/Qbox-project/qbx_towjob/assets/26857315/2529b321-f8da-45bd-8765-acedb6d1d33c)

Reply:
That makes no sense as to why they were removed

Reply:
Yeah well, that's how it was by default, I didn't change that.